src/SpoolmanManager.cpp

```
@@ -1244,6 +1244,63 @@ bool SpoolmanManager::lookupSpoolByUid(const char* uid, SpoolDetails& outDetails
+        return 0.0f;
+    }
+
+    int spoolId = findSpoolByUuidGlobal(uid);
+    if (spoolId < 0) {
+        Serial.printf("SpoolmanManager: deductFromSpoolman — spool not found for %s\n", uid);
+        xSemaphoreGive(httpMutex_);
+        return 0.0f;
+    }
+
+    // Get current remaining weight
+    char path[64];
```

Clang (14.0.6)

[warning] do not declare C-style arrays, use std::array<> instead

(modernize-avoid-c-arrays)